### ISSUE TITLE
[Feature Request] Avoid calling clearTempFiles() by default while export()

### ISSUE THREAD

**Issue by albilu** [SeunMatt/mysql-backup4j#33]
https://github.com/SeunMatt/mysql-backup4j/blob/c01932c42ef0ef7b7e4917260a1d855d5291db09/src/main/java/com/smattme/MysqlExportService.java#L458

**Use Case:**
We need to be able to grab the exported files(specifically the zip) use them in some process (example send them to an SFTP server) and then call clearTempFiles() to delete the files.

Right now this is not possible as clearTempFiles() is call by default while exporting and if i set PRESERVE_GENERATED_ZIP i can use the zip file as i want but then when i called clearTempFiles() the files are not cleaned.

This will give more flexibility. So please take into account.

**Comment by SeunMatt** [comment]
This is already possible.

```
//...
properties.setProperty(MysqlExportService.PRESERVE_GENERATED_ZIP, "true");
MysqlExportService mysqlExportService = new MysqlExportService(properties);
mysqlExportService.export();
File file = mysqlExportService.getGeneratedZipFile();
```

You can read more about it here https://smattme.com/posts/how-to-backup-mysql-database-programmatically-using-mysql-backup4j/

**Comment by albilu** [comment]
> This is already possible. Set the property `MysqlExportService.PRESERVE_GENERATED_ZIP` to `true`.
> 
> ```
> //...
> properties.setProperty(MysqlExportService.PRESERVE_GENERATED_ZIP, "true");
> MysqlExportService mysqlExportService = new MysqlExportService(properties);
> mysqlExportService.export();
> File file = mysqlExportService.getGeneratedZipFile();
> ```
> 
> You can read more about it here https://smattme.com/posts/how-to-backup-mysql-database-programmatically-using-mysql-backup4j/

This is not available anymore in version 1.2.1 wich is the latest.

You can verify from the master branch also the method doesnt take any parameter: https://github.com/SeunMatt/mysql-backup4j/blob/master/src/main/java/com/smattme/MysqlExportService.java#L469

From the link you provided the boolean parameter was indeed supported for 1.0.1

Was there any reason to remove it? Otherwise it is a regression and you might want to add it back
